apps/condo/domains/billing/utils/testSchema/index.js

```diff
@@ -734,6 +746,25 @@ function createTestRecipient (extra = {}) {
     }
 }
 
+
+async function makeClientWithResidentAndServiceConsumer (property, billingAccount, organization = null) {
```

Comment:
Did we really create all the residents and consumers manually? :/

Reply:
It would be really nice if you refactored tests in BillingReceipt and alike :-)

Reply:
we created them separately
and some times we are creating staff user and change it later to resident - which is not correct, as this client will have an Organization